src/wallet/types.ts

```diff
@@ -161,6 +161,12 @@ export interface Utxo {
   addressPath: string; // path to generate output address
 }
 
+export interface AuthorityTxOutput {
```

Comment:
Should we have which kind of authority this is?

Reply:
That's a nice refactor, done in b033fd954ad7715f7dc82c775082ba3dbf5b53cc